message.go

```
@@ -449,6 +449,32 @@ func (r RedisResult) CachePXAT() int64 {
 	return r.val.CachePXAT()
 }
 
+// String returns human-readable representation of RedisResult
+func (r *RedisResult) String() string {
+	v, err := (*prettyRedisResult)(r).MarshalJSON()
+	if err != nil {
```

We can just ignore the err here.

```suggestion
```